packages/fast-animation/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+
```

Comment:
please do not make any changes to CHANGELOG.md, these are all generated by Lerna, if we want to make any changes we should have a task to investigate changing headers etc.